app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -395,8 +398,8 @@ public void onSaveInstanceState(@NonNull final Bundle bundle) {
 
     @Override
     public void reloadContent() {
-        if (!TextUtils.isEmpty(searchString)
-                || (searchEditText != null && !TextUtils.isEmpty(searchEditText.getText()))) {
+        if (!TextUtils.isEmpty(searchString) || (searchEditText != null
+                && !isBlank(searchEditText.getText().toString()))) {
```

Comment:
I think a helper method for ``isBlank(searchEditText.getText().toString())`` would be good because it's used 3 times here.

Maybe something like
```java
public static boolean isTextViewBlank(final TextView textView) {
  return Utils.isBlank(textView.getText().toString());
}
```

Reply:
Or even non static and without the parameter